BUILDING.md

```diff
@@ -65,22 +72,29 @@ The build output will be in the directory you specified with `do config`.
 
 ### Install Android SDK
 * Unzip the [Android SDK](https://dl.google.com/android/repository/tools_r25.2.3-macosx.zip) to a directory of your choosing.
+  * Instead, if you have Homebrew, you can install via `brew install android-sdk` and the default installation location would be `$HOMEBREW_PREFIX/opt/android-sdk`.
```

Comment:
`$HOMEBREW_PREFIX` does not appear to be a automatically defined environment variable. You *could* suggest:

```echo `brew config | grep HOMEBREW_PREFIX | awk '{print $2}'`/opt/android-sdk```

or something, but I'd probably just suggest something like:

  * Instead, if you have Homebrew, you can install via `brew install android-sdk` and the default installation location would be *HOMEBREW_PREFIX*`/opt/android-sdk`, where *HOMEBREW_PREFIX* can be found using `brew config`.

It's also worth mentioning that `do` does not currently support resolving env vars.
Alternatively to this, we could just make do search for the brew location to fill it in as a suggestion...